javascript/javascript2/week1/JS2W1.js

```diff
@@ -0,0 +1,82 @@
+////////////////////////////////////////////////////////////Find the shortest word
+
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+const number = [];
+
+function findTheShortest(words) {
+    for (let i = 0; i < words.length; i++) {
```

Comment:
I really like you solution but do we really need a new array, can we use loop and iterate the array and compare length from one to another